README.md

```diff
@@ -134,37 +134,7 @@ The versioning in Filecoin-FFI currently follows the Lotus versioning.  For exam
 
 #### Release Process
```

Comment:
Can we also have a note about if/how/when release assets are created (since I don't think that is covered in https://github.com/filecoin-project/filecoin-ffi/pull/464 )